tasks/post-to-cmr/index.js

```diff
@@ -71,10 +64,40 @@ async function getMetadata(xmlFilePath) {
   // Currently, only supports files that are stored on S3
   const parts = xmlFilePath.match(/^s3:\/\/(.+?)\/(.+)$/);
   const obj = await getS3Object(parts[1], parts[2]);
-
   return obj.Body.toString();
 }
 
+/**
+ * returns a list of CMR xml files
+ *
+ * @param {Array} input - an array of s3 uris
+ * @param {string} granuleIdExtraction - a regex for extracting granule IDs
+ * @returns {Promise<Array>} promise resolves to an array of objects
+ * that includes CMR xmls uris and granuleIds
+ */
+async function getCmrFiles(input, granuleIdExtraction) {
+  const files = [];
+  const expectedFormat = /.*\.cmr\.xml$/;
+
+  for (const filename of input) {
+    if (filename.match(expectedFormat)) {
```

Comment:
When would we get a filename that does not match the expected format? What should we do then?

Reply:
In this case we're making an array of the .cmr.xml files, so with other files (like .hdf or .jpg) this will ignore those to get just the cmr files.

This was also existing code that got moved around a little. I wonder if that regex should be configurable or if the `.cmr.xml` suffix is a common/standard naming scheme.